ledger/txtail.go

```diff
@@ -207,6 +207,11 @@ func (t *txTail) putLV(lastValid basics.Round, id transactions.Txid) {
 }
 
 func (t *txTail) getBlockTimeStamp(rnd basics.Round) (int64, error) {
+	// allow timestamps in a first block
+	if rnd == 0 {
+		return 0, nil
+	}
```

Comment:
that's fine, but consider moving this to after the 
```golang
       if r, ok := t.recent[rnd]; ok {
 		return r.timestamp, nil
 	}
```

for performance reasons. ( i.e. we would rarely get here with rnd==0, so we should test that later on. )

Reply:
done